concent-builder/build-test-and-push.yml

```diff
@@ -64,12 +64,12 @@
               --network {{ cluster }}-{{ image_prefix }}unittest-network                                         \
               --name {{ cluster }}-{{ image_prefix }}postgresql-unittest {{ image_prefix }}postgresql:latest
 
-        - name:    Run concent-api container and django unit tests
-          command: docker run                                                                                                                                 \
-              --rm                                                                                                                                            \
-              --volume {{ build_dir }}/concent-deployment/kubernetes/build/run-unit-tests.sh:/usr/local/bin/run-unit-tests.sh:ro                              \
-              --volume {{ build_dir }}/concent-deployment/kubernetes/build/docker_testing.py:/srv/http/concent_api/concent_api/settings/docker_testing.py:ro  \
-              --network {{ cluster }}-{{ image_prefix }}unittest-network concent-concent-api:latest /usr/local/bin/run-unit-tests.sh
+   #     - name:    Run concent-api container and django unit tests
+   #       command: docker run                                                                                                                                 \
+   #           --rm                                                                                                                                            \
+   #           --volume {{ build_dir }}/concent-deployment/kubernetes/build/run-unit-tests.sh:/usr/local/bin/run-unit-tests.sh:ro                              \
```

Comment:
Why is this commented out?

Reply:
It was accidentally committed. I know this, I will remove it before merging.